src/lib.rs

```diff
@@ -59,23 +64,31 @@ pub struct Record {
     pub record_number: u64,
     /// Classification of the record.
     pub classification: Classification,
-    /// Whether the alarm function of the CUAS system is active or not.
+    /// (Optional)
     /// An Alarm is defined as the function of a CUAS system alerting an Operator via the HMI and
     /// the generation of associated data in the UAS Activity Log, as a result of Declared UAS
     /// activity.
```

Comment:
Now that we have an Alarm type, this describes the type better than the member.

Reply:
That is true, but I don't understand what differences there should be between member descriptions and type descriptions in situations like this.